Automattic/wp-calypso
# Hosting import flow: allow picking an existing site

Closes https://github.com/Automattic/dotcom-forge/issues/2640.

## Proposed Changes

The user should be able to pick an existing site as a target for the migration.

## Testing Instructions

In an account with zero sites, check that we're creating one in the background after confirming that we want to import an existing site.

In an account with existing sites, check that the site picker is shown after agreeing to import an existing site. Check that it's possible to use an existing site, as well as create a new one by clicking the "create a new site" links.
